### PR TITLE
ref: Mark NVIDIA source as `accept_invalid_certs`

### PR DESCRIPTION
**Pull request by loewenheim**
This sets the `accept_invalid_certs` flag on the NVIDIA symbol source. This flag was defined on the Symbolicator side in https://github.com/getsentry/symbolicator/pull/1405.

Background: we currently can't fetch symbols from the NVIDIA source because we can't verify its SSL certificates. We've added the capacity to skip SSL cert verification to Symbolicator, and this PR enables it for the NVIDIA source.

The epic is tracked in https://github.com/getsentry/symbolicator/issues/1402.

This is intended as a stopgap until we manage to verify the certificates one way or another.

### CODE REVIEW

**Review by anonrig — APPROVED** [review]

File: src/sentry/conf/server.py [hunk 1]
```diff
@@ -3302,6 +3302,7 @@ def build_cdc_postgres_init_db_volume(settings: Any) -> dict[str, dict[str, str]
         "filters": {"filetypes": ["pe", "pdb"]},
         "url": "https://driver-symbols.nvidia.com/",
         "is_public": True,
+        "accept_invalid_certs": True,
```
Comment (anonrig) [inline src/sentry/conf/server.py:3305]: Can we add a TODO here?

**Review by anonrig — APPROVED** [review]

File: src/sentry/conf/server.py (hunk 1 above)
Comment (loewenheim) [reply src/sentry/conf/server.py:3309]: Good idea. What do you think it should say? That we want to remove it again when the underlying problem is fixed?

Comment (anonrig) [reply src/sentry/conf/server.py:3313]: Yeah something like that. Maybe a definition of why we had this issue in the first place. In the future the context might be lost and we might have to dig deep into Github in order to find the root cause.

Comment (anonrig) [reply src/sentry/conf/server.py:3313]: `once the cert problem is cleared up` meaning when CURL supports XXX certification feature. might be a better description with a more searchable comment

Comment (loewenheim) [reply src/sentry/conf/server.py:3315]: I made the comment more specific. @Swatinem to your knowledge, is this an accurate description of the problem?

**Review by Swatinem — APPROVED** [review]
sounds perfect

**Pull request merged**